interface-definitions/interfaces-bridge.xml.in

```diff
@@ -178,6 +217,14 @@
               <valueless/>
             </properties>
           </leafNode>
+          <leafNode name="vlan">
```

Comment:
Why is this node required when we already have the vif nodes to include?

Reply:
The comment of the switch indicates that once the user sets it, the vlan filter processing function of the bridge will be enabled, and the bridge will become a vlan-aware bridge

Reply:
Thy don't we make this auto recognized by the Python code once vif or vif-s or allowed-vlan or native-vlan option is set?

Reply:
Do you want to automatically enable vlan filter in this case instead of letting users specify?

Reply:
If a user configures all the required stuff for vlan-aware bridging we can safely assume he want's to have vlan aware bridging.

Reply:
I agree, and have modified the implementation